specs_e2e/rails_6_1/test.sh

```
@@ -0,0 +1,57 @@
+rm -vf test/playwright/e2e/rails_examples/using_vcr.cy.js
+
+echo '-- playwright run'
+cd test
+cp -fv ../../playwright.config.js .
+npx playwright install-deps
+npx playwright install
+npx playwright test test/playwright
+# npx playwright show-report
+
+echo '-- stop rails server'
+kill -9 `cat ../../server.pid` || true
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)